build.gradle

```diff
@@ -55,8 +55,10 @@ configure(subprojects.findAll { it.path.startsWith(':sdk:') && it.path.count(':'
         packagingOptions {
             exclude 'META-INF/DEPENDENCIES'
             exclude 'META-INF/LICENSE'
+            exclude 'META-INF/LICENSE.md'
             exclude 'META-INF/license'
             exclude 'META-INF/NOTICE'
+            exclude 'META-INF/NOTICE.md'
             exclude 'META-INF/notice'
             exclude 'META-INF/ASL2.0'
             exclude("META-INF/*.md")
```

Comment:
Odd, because this seems like it should be excluding `*.md`.

Reply:
I'm unable to validate this but we know for beta-1 *.md didn't work. let's sync offline.

Reply:
Discussed offline. Its the singing phase adding these entries to ARR hence we can't validate it locally at the moment. 
It won't hurt adding those entries anyway, it either works or we can request users to exclude it for noe (until we have a way to validate it through staging)